index.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+   <link rel="stylesheet" type="text/css" href="./css/style.css">
+   <meta charset="UTF-8">
+   <meta name="viewport" content="width=device-width, initial-scale=1.0">
+   <title>Portfolio Web Page </title>
+</head>
+<body>
+   <div id="divbar">
+      <a href="#welcome-section">Contact</a>
+      <a href="#projects">Work</a>
+      <a href="#profile-link">About</a>
+   </div>

```

Comment:
Kindly use nav instead of div 
I think NAV is better because it's a HTML5 element indicating to your browser that its a navigation section.

Reply:
Awesome ! :trophy: